testutils/testutils-kmp/src/commonMain/kotlin/androidx/kruth/ComparableSubject.kt

```diff
@@ -37,4 +37,17 @@ open class ComparableSubject<T : Comparable<T>> constructor(actual: T?) : Subjec
             fail("Expected to be less than $other, but was $actual")
         }
     }
+
+    /**
+     * Checks that the subject is greater than or equal to [other].
+     *
+     * @throws NullPointerException if [actual] or [other] is `null`.
+     */
+    fun isAtLeast(other: T?) {
+        if (actual == null || other == null) {
```

Comment:
Let's use the existing [requireNonNull](https://github.com/androidx/androidx/blob/59a572d744be4366849cb0164bc834c2b44e88ea/testutils/testutils-kmp/src/commonMain/kotlin/androidx/kruth/Utils.kt#L27) function?

Reply:
@arkivanov Replacing this line with `requireNotNull` would mean an `IllegalArgumentException` would be thrown instead of a `NullPointerException`.  This diverges from Truth, as their `isAtLeast` function [also throws a `NullPointerException`](https://github.com/google/truth/blob/19dcd43928041722b8f1bd79bcd4da4faea89426/core/src/test/java/com/google/common/truth/ComparableSubjectTest.java#L61-L65). Was the original suggestion to also change the throwable type?

Reply:
I meant our internal function, see https://github.com/androidx/androidx/blob/59a572d744be4366849cb0164bc834c2b44e88ea/testutils/testutils-kmp/src/commonMain/kotlin/androidx/kruth/Utils.kt#L27

Reply:
Oops, my bad! You had linked it in the first message — I completely overlooked it.

Changed in https://github.com/androidx/androidx/pull/519/commits/20d36a60ff156846bd7271377b03cab5328940da.